src/components/Request/index.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react'
+import {startsWith} from 'lodash'
+import {logout} from 'utils/authentication'
+import {default as RequestBase} from './components/Request'
```

Comment:
@tayiorbeii I would agree if the base component was going to stay in this project, but it is now in `egghead-ui` and once `egghead-ui` published a new version, I'll be deleting it from this project so it will be `import {Request as RequestBase} from 'egghead-ui'` and then wrapping it with project defaults like this.